docs/reference/querying.md

```diff
@@ -0,0 +1,108 @@
+# Sending queries to Elasticsearch
+
+The Python Elasticsearch client provides several ways to send queries to Elasticsearch. This document explains the details of how to construct and execute queries using the client. This document does not cover the DSL client.
```

Comment:
I have removed all references to a "DSL client" to avoid confusion, and instead used "DSL module", now that the clients are unified.